src/components/WebexMemberRoster/WebexMemberRoster.scss

```diff
@@ -0,0 +1,7 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-member-roster {
+  .section-title {
+    @include body-fonts;
+    color: $md-gray-40;
+    border-bottom: 0.1rem solid $md-gray-30;
```

Comment:
@lalli-flores there are some issues when using rems for border widths, is this a hard requirement? https://bugs.chromium.org/p/chromium/issues/detail?id=382483

Reply:
actually this links to an old (and fixed) bug, but still, as the browsers can only render integer widths, doesn't it make more sense to specify the border as 1px, instead of 0.0625rem? (which is basically converting 1px in rems)

If 1rem==31px, the border is still rendered one pixel wide (in chrome), but if 1rem==32px, the border will be 2 pixels, is this the intended behavior?

Reply:
The intended behavior is for all things to scale accordingly. I would prefer to maintain rem units across CSS, less we run into issues.